src/ExtensionsSample/Samples/DocumentDBSamples.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using ExtensionsSample.Models;
+using Microsoft.Azure.WebJobs;
+
+namespace ExtensionsSample.Samples
+{
+    // To use the DocumentDB samples:
+    // 1. Create a new DocumentDB account
+    // 2. Add the DocumentDB Endpoint Uri to the 'AzureWebJobsDocumentDBUri' App Setting in app.config
+    // 3. Add the DocumentDB Primary Key to the 'AzureWebJobsDocumentDBKey' App Setting in app.config
+    // 4. Add typeof(DocumentDBSamples) to the SamplesTypeLocator in Program.cs
+    public static class DocumentDBSamples
+    {
+        // POCO Output binding
+        //   The binding inserts the newly created item into the specified Collection when the
+        //   method successfully exits.
+        //   If the database or the collection do not exist, they are automatically created.
```

Comment:
False, by design. Document Collections are *billable* entities so the DocDB team recommends this is opt-in.

---
In reply to: [56354011](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/47#discussion_r56354011) [](ancestors = 56354011)